Gemfile

```diff
@@ -5,7 +5,7 @@ gemspec
 
 group :test do
   gem 'actionpack'
-  gem 'logstash-event', '1.1.5'
+  gem 'logstash-event', git: 'https://github.com/elasticsearch/logstash.git', tag: '1.5'
```

Comment:
Good point.
Explanation:
logstash does not release any gems on rubygems, but they have two gemspecs within their repo.

I wanted to be sure that the right tag of the surrounding repo matches the current state of the 1.3 version of logstash-event. If you have any idea who this could be done in a better way please show me. 

I can add a comment in the gemspecs if you want to.